openedx/core/djangoapps/user_api/urls.py

```diff
@@ -4,7 +4,7 @@
 
 from .accounts.views import AccountView
 from .preferences.views import PreferencesView, PreferencesDetailView
-
+from ..profile_images.views import ProfileImageView
 from django.conf.urls import patterns, url
```

Comment:
Nit: I believe somewhere in our conventions it says the import order should have the local imports on the bottom and external imports on top.